src/about.js

```diff
@@ -0,0 +1,67 @@
+function setAboutPage() {
+
+  const aboutDiv = createElementWrapper("div", ["card", "mb-3"])
```

Comment:
another approach i can recommended here, because the creation of the elements and classes is long
you can also create is as DOM element hidden, retrieve it from the dom duplicate( copy ) the element, modify it and append it to the DOM.